app/sampledata/models/cocktail_glass.mtl

```diff
@@ -0,0 +1,10 @@
+# WaveFront *.mtl file (generated by CINEMA 4D)
```

Comment:
@shahafhermann nice! BTW, you may try 'Figma' website for a much easier design flow- it's almost like Photoshop, but you get an XML representing your design in the end

Reply:
I'll check it out but I don't think it would help since we need actual 3D models. These models are free assets I found on some platforms...

Reply:
OMG, this is so cool